src/desktop.md

```diff
@@ -275,86 +290,113 @@ $ flutter run -d linux
   the `-d <platform>` tag is optional.
 {{site.alert.end}}
 
-### Build a release app
+## Build a release app
 
-To generate a release build run one of the following commands:
+To generate a release build run one of the
+following commands:
 
 ```terminal
 $ flutter build windows
 $ flutter build macos
 $ flutter build linux
 ```
 
-### Distribution
-
-**In general, we don't recommend releasing a desktop app until
-desktop support is stable.**
-There are not yet full instructions, or tooling support,
-for making distributable applications. However,
-here is some information about how to use the current
-build output on other machines for testing purposes.
-
-#### Windows
-
-For building Windows executables, you can either use tooling to construct an
-MSIX installer, or you can build your own zip file that collects
-the components together.
-
-##### MSIX Packaging
-
-[MSIX][] is Microsoft's Windows app package format that provides a modern 
-packaging experience to all Windows apps. This format can either be used 
-to ship applications to Microsoft's Windows Apps store, or distribute 
-application installers directly.
-
-The easiest way to create an MSIX distribution for a Flutter project is to use
-the [`msix` pub package][msix package]. For an example of using the `msix` package
-from a Flutter desktop app, see the [Desktop Photo Search sample][].
+## Distribution
+
+**We don't recommend releasing a desktop
+app until desktop support is stable,**
+however, here is some information that you
+might still find useful.
+
+### Windows
+
+There are various approaches you can use for
+distributing your Windows application.
+Here are some options:
+
+* Use tooling to construct an MSIX installer
+  for your app and distribute it through
+  the Microsoft Windows App Store.
+  You don't need to manually create a signing
+  certificate for this option as it is
+  handled for you.
+* Construct an MSIX installer and distribute
+  it through your own website. For this
+  option, you need to to give your app a
+  digital signature in the form of a
+  `.pfx` certificate.
+* Collect all of the necessary pieces
+  and build your own zip file.
+  (FIX: Does this option use the .pfx
+  certificate or is signing not needed?)
```

Comment:
Is this FIX line meant to still be here?

Reply:
@csells do you know the answer to this one, or know who might? If not, I can hunt down the answer here, or find someone to do so.

Reply:
I don't know. I know @domesticmouse has been looking at this from the Windows Store pov but I don't know if he's also look at it from the direct distribution pov.

Reply:
My understanding is that the .pfx file is used by the machinery that installs a MSIX distribution. If you are just copying over a zip file, it should just work. I'll try it later today, once I can generate a clean Windows machine.